pandas/io/parsers.py

```diff
@@ -825,6 +825,22 @@ def _validate_usecols_arg(usecols):
     return usecols
 
 
+def _validate_parse_dates_arg(parse_dates):
```

Comment:
Existing code and logic does not validate IINM.  For example, line 298 is about overriding `parse_dates` if it is a boolean and `date_parser` is passed.